stellargraph/core/graph_networkx.py

```diff
@@ -772,6 +772,30 @@ def to_adjacency_matrix(self, nodes: Optional[Iterable] = None):
             self._graph, dtype="float32", weight=self._edge_weight_label, format="coo"
         )
 
+    def to_networkx(self):
+        # Despite this class using NetworkX, this implementation does not directly use that
+        # representation, so that it can be reused as we move away from being NetworkX-based.
+        if self.is_directed():
+            graph = nx.MultiDiGraph()
+        else:
+            graph = nx.MultiGraph()
+
```

Comment:
why is it always converted to multi(di)graph?

Reply:
I chose this because `StellarGraph` itself allows for multiple edges (and, the old `StellarGraph` types inherited from `nx.MultiGraph`/`nx.MultiDiGraph`). I don't think we record or check that edges are unique anywhere, so doing the check here might be slightly expensive.